webui/elm/src/View.elm

```diff
@@ -151,9 +151,9 @@ docsLink =
         , HA.style "border-top" "4px solid rgba(0, 0, 0, 0)"
         ]
         [ a
-            [ class "no-underline hover:text-white font-bold flex items-center"
+            [ class "no-underline hover:text-white flex items-center"
             , HA.target "_blank"
-            , href "/docs"
+            , href "/docs/"
```

Comment:
question: I think you fixed it so it can be without the trailing slash. Is this because `swagger-ui` requires a slash and to keep them consistent?

Reply:
as I was reading up on setting up swagger and hosting it's UI I had to fix some relative links and having the trailing slash is the difference between the relative links working off the correct base vs not. so it seems it's best if these type of external resource links have the slash :thinking: